ruby-gem/libconfigure.gemspec

```diff
@@ -0,0 +1,29 @@
+require_relative 'version'
+
+Gem::Specification.new do |s|
+  s.name        = 'libconfigure'
```

Comment:
Given this will likely be published on RubyGems's public registry, I wonder if we can't find a more descriptive name:
 - Not sure why you chose `lib*` prefix given the gem ends up bundling binaries so it's more about distributing binaries than a library
 - `configure` is quite generic. It makes sense in the context of a8c because we all know what it means (tho… that's debatable: in Apps and mobile teams, we know, but outside this circle, like in the rest of a8c, probably not, and the name isn't really descriptive enough to make it clear or discoverable without reading the README and understanding the context)

It's probably too late to change the name of the native (Rust) tool (or… is it?) to use the occasion of this refactoring to come up with a better name, but maybe it's still ok to come up with a more descriptive name for the ruby gem at least? What about something like `configure-secrets` or `secrets-configure` or `manage-secrets`?

Reply:
We should publish this in the public repo, yes – this will likely live as a release toolkit dependency for a good long while, and may eventually be used for CI system management as a puppet plugin as well.

Regarding naming, I'm not set on anything in particular – I went with the `lib` prefix to indicate that this is distributing the `libconfigure` target from the Rust project. 

`configure-secrets` is an ok fallback if we can't think of something more exciting 😁

Reply:
Given the limited scope of this gem, because there's no plan to advertise it currently, we might as well go with this name and update it to something we may feel more appropriate in the future

Reply:
I think it might be more difficult and not recommended to yank an old gem and publish a new one (under the new name) as opposed to choosing a right name right away though. Avoids having to go thru the whole deprecation + yanking + redirect hassle.

Some ideas:
 - `a8c-secrets-configure`
 - `secrets-config` _(my personal preference so far)_
 - `secrets-configure`
 - `configure-secrets`
 - …